.vscode/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "label": "build",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/ikspoz.sln",
```

Comment:
Changed this to build the `.sln` instead of the specific project to make this future proof should we start adding more projects.